packages/hdwallet-ledger/src/utils.ts

```diff
@@ -96,503 +95,277 @@ export const createXpub = (
 };
 
 type NetworkMagic = {
-  apiName: string;
-  unit: string;
-  name: string;
-  appName?: string;
-  satoshi?: number;
   bitcoinjs: {
-    bech32?: string;
     bip32: {
-      private?: number;
       public: Partial<Record<core.BTCInputScriptType, number>>;
     };
-    messagePrefix: string;
-    pubKeyHash?: number;
-    scriptHash?: number;
-    wif?: number;
```

Comment:
dangerous removal, please revert bitcoinjs changes. we are passing this payload to bitcoinjs for transaction building which is using at least `bech32`, `pubKeyHash` and `scriptHash` at first glance and likely others.

Happy to address this in a follow up PR to update ledger to use modern psbt and pull out bitcoinjs network configs into a base package for use against any packages using bitcoinjs-lib.

would be fine with removal of the unused values outside of bitcoinjs being removed at this time

Reply:
Cleaned diff up to the bare minimum this is bringing in https://github.com/shapeshift/hdwallet/pull/670/files#diff-48c2229f8d069d68109d331ff81dba7fc4d501ae143cb400cae2b543dda1d119 - @woodenfurniture FYI ^^